CHANGELOG.md

```diff
@@ -1,15 +1,199 @@
 # Rerun changelog
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
+The biggest news is the ability to create a _blueprint_ via the Python logging API:
+
+```py
+import rerun.blueprint as rrb
+
+blueprint = rrb.Blueprint(
+    rrb.Horizontal(
+        rrb.Spatial3DView(origin="/world"),
+        rrb.Grid(
+            rrb.BarChartView(name="Word embedding", origin="/embedding"),
+            rrb.TimeSeriesView(name="Loss", origin="/loss"),
+        ),
+        column_shares=[2, 1],
+    ),
+    rrb.TimePanel(expanded=False),
+)
+```
+
+The blueprint can then be sent to the viewer with
+```py
+rr.send_blueprint(blueprint)
+```
+
+Or stored to file, and the later opened in the viewer:
+```py
+blueprint.save("my_nice_dashboard.rbl")
+```
+
+Right now this is only supported by the Python API, with C++ and Rust support coming later.
```

Comment:
Maybe we should explicitly communicate that it's already possible today to log data from C++ or Rust all the while logging a blueprint for python.